package.json

```diff
@@ -54,10 +54,17 @@
         "babelify",
         {
           "presets": [
-            "es2015",
-            "stage-0"
+            ["env", {
```

Comment:
I think we might be better off removing the options here and having it default to using what's in the `.babelrc`

Reply:
Seems like Babelify doesn't recognise .babelrc anymore with babel@6, so there wouldn't actually be any need to update .babelrc, but the package.json file is still necessary. I think we still should keep .babelrc updated, incase it could have other uses in the future.

https://github.com/babel/babelify/issues/151

https://github.com/babel/babelify/issues/151